sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallingServerAsyncClient.java

```diff
@@ -310,6 +310,32 @@
         }
     }
 
+    /**
+     * Answer a Call
+     *
+     * @param incomingCallContext The incoming call context.
+     * @throws CallingServerErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return Response for a successful answer request.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<CallConnectionAsync> answerCall(
```

Comment:
we need an answercallwithresponse too, correct?

Reply:
added